bfasst/status.py

```diff
@@ -97,18 +97,25 @@ class MapStatus(enum.Enum):
 
 
 class BfasstException(Exception):
-    def __init__(self, err, msg):
+    def __init__(self, err, msg, creator=None):
```

Comment:
Can you add a comment here explaining this argument?

Reply:
Done